src/index.css

```
@@ -1,15 +1,64 @@
+
+@layer base {
+  :root {
+    --radius: 12px;
+
+    /* Brand Colors (HSL format) */
+    --primary: 146 50% 36%;   /* #2e8b57 */
+    --primary-foreground: 0 0% 100%;
+    --accent: 47 89% 60%;     /* #f4c430 */
+    --accent-foreground: 222 47% 11%;
 
+    --background: 0 0% 100%; 
```

Stylelint (17.2.0)

[error] Unexpected empty line before custom property (custom-property-empty-line-before)

(custom-property-empty-line-before)